contrib/inventory_builder/tests/test_inventory.py

```diff
@@ -246,25 +246,25 @@ def test_set_k8s_cluster(self):
 
         self.inv.set_k8s_cluster()
         for host in expected_hosts:
-            self.assertTrue(
-                host in
+            self.assertIn(
```

Comment:
Same suggestion as the first comment

`self.assertTrue(set(expected_hosts).issuperset(self.inv.yaml_config['all']['children'][group]['children']))`